l10n_ar_edi_ux/wizards/res_partner_update_from_padron_wizard.py

```diff
@@ -169,8 +173,17 @@ def _update(self):
     def automatic_process_cb(self):
         for partner in self.partner_ids:
             self.partner_id = partner.id
-            self.change_partner()
-            self._update()
+            try:
+                self.change_partner()
+                self._update()
+            except (UserError) as exp:
+                patron = re.compile(r"'error': \['(.*?)'\]")
+                if resultado := patron.search(str(exp)):
+                    error = resultado.group(1)
+                else:
+                    error = str(exp)
+                partner.message_post(body="Falló actualización AFIP: "+ error)
```

Comment:
crear un nuevo metodo que se llame algo como procesar_error donde coloquemos las lineas que procesando la info del error que estan repetidos

```suggestion
                self.process_error(exp)
```
```python
def process_error(self, exp):
       patron = re.compile(r"'error': \['(.*?)'\]")
                if resultado := patron.search(str(exp)):
                    error = resultado.group(1)
                else:
                    error = str(exp)
                partner.message_post(body="Falló actualización AFIP: "+ error)```